bower.json

```diff
@@ -37,7 +37,8 @@
     "vaadin-overlay": "vaadin/vaadin-overlay#^3.2.0",
     "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.1.0",
     "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.1.0",
-    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.0.0"
+    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.0.0",
+    "vaadin-icons": "vaadin/vaadin-icons#^4.3.1"
```

Comment:
The component itself doesn't need "vaadin-icons" runtime. It's used in the demos only so this should be moved to "devDependencies"

Reply:
Done